### PR TITLE
Generated a list of logs for the bootstrap command

### PR DESCRIPTION
**Pull request by ishitasequeira**
**What type of PR is this?**
 /kind design

**What does does this PR do / why we need it**:
Improve the bootstrap output. Made the output more descriptive.

Fixes #?
https://issues.redhat.com/browse/GITOPS-211

**How to test changes / Special notes to the reviewer**:
Co-authored by @gaganhegde  and @ishitasequeira

### CODE REVIEW

File: pkg/pipelines/bootstrap.go [hunk 1]
```diff
@@ -311,3 +318,8 @@ func checkPipelinesFileExists(appFs afero.Fs, outputPath string, overWrite bool)
 	}
 	return nil
 }
+
+//logs Success message to the outputs
+func logBootstrapStatus(message string) {
+	log.Success(message)
```
Comment (chetan-rns) [inline pkg/pipelines/bootstrap.go]: Using `log.Success` everywhere may not be a good idea. Usually, success messages will be shown in the end. We have to follow the conventions used in other `odo` commands

File: pkg/pipelines/bootstrap.go [hunk 2]
```diff
@@ -107,6 +111,7 @@ func bootstrapResources(o *BootstrapOptions, appFs afero.Fs) (res.Resources, err
 	appName := repoToAppName(repoName)
 	serviceName := repoName
 	secretName := secrets.MakeServiceWebhookSecretName(ns["dev"], serviceName)
+	logBootstrapStatus("Bootstrapping dev and stage environment resources")
```
Comment (chetan-rns) [inline pkg/pipelines/bootstrap.go]: We can't use hardcoded environment names because of prefix

**Comment by chetan-rns** [comment]
Should we add the logs inside the builder functions Environment() and Service()?
Something like:
```
func (b *builder) Environment(env *config.Environment){
    .......
    log.Infof("Generating %s environment",env.Name)
}
```

**Comment by chetan-rns** [comment]
> Should we add the logs inside the builder functions Environment() and Service()?
> Something like:
> 
> ```
> func (b *builder) Environment(env *config.Environment){
>     .......
>     log.Infof("Generating %s environment",env.Name)
> }
> ```

Doing this may cause issues in other commands. So ignore this

File: pkg/pipelines/bootstrap.go [hunk 3]
```diff
@@ -311,3 +316,8 @@ func checkPipelinesFileExists(appFs afero.Fs, outputPath string, overWrite bool)
 	}
 	return nil
 }
+
+//logs Success message to the outputs
+func logBootstrapStatus(message string) {
```
Comment (chetan-rns) [inline pkg/pipelines/bootstrap.go]: Any advantage of using this function?

File: pkg/pipelines/bootstrap.go [hunk 4]
```diff
@@ -141,6 +145,7 @@ func bootstrapResources(o *BootstrapOptions, appFs afero.Fs) (res.Resources, err
 	}
 	secretFilename := filepath.Join("03-secrets", secretName+".yaml")
 	secretsPath := filepath.Join(config.PathForPipelines(cfg), "base", secretFilename)
+	logBootstrapStatus(fmt.Sprintf("Webhook secret to authenticate pull/push to %s repo created sucessfully", serviceName))
```
Comment (chetan-rns) [inline pkg/pipelines/bootstrap.go]: We could use `log.Infof` for formatting instead of `fmt.Sprintf`

File: pkg/pipelines/bootstrap.go (hunk 2 above)
Comment (gaganhegde) [reply pkg/pipelines/bootstrap.go]: This has been addressed by taking into consideration the prefix.

File: pkg/pipelines/bootstrap.go (hunk 1 above)
Comment (gaganhegde) [reply pkg/pipelines/bootstrap.go]: log.info in accordance with other odo commands has been incorporated.

File: pkg/pipelines/bootstrap.go (hunk 3 above)
Comment (gaganhegde) [reply pkg/pipelines/bootstrap.go]: Just looks more aesthetic while logging errors!

File: pkg/pipelines/bootstrap.go (hunk 4 above)
Comment (gaganhegde) [reply pkg/pipelines/bootstrap.go]: Will make the changes:)

Comment (gaganhegde) [reply pkg/pipelines/bootstrap.go]: The changes have been addressed

File: pkg/odo/cli/pipelines/bootstrap.go [hunk 5]
```diff
@@ -78,7 +78,7 @@ func (io *BootstrapParameters) Run() error {
 	if err != nil {
 		return err
 	}
-	log.Success("Success")
+	log.Success("Bootstraped Successfully")
```
Comment (bigkevmcd) [inline pkg/odo/cli/pipelines/bootstrap.go]: s/Bootstraped/Bootstrapped/

**Pull request closed without merging**